Minor changes

kevinf100 opened the pull request:
VS Code complained one of the hex colors wasn't right.
Added a feature that will ask to user to confirm resetting tree(s) before doing so.
Changed Epoch url as <del>https://www.project-epoch.net/</del> was killed by Blizzard RIP.
Added GitHub source link to Header.

kevinf100 committed:
fix: hex color one length short

kevinf100 committed:
feat: added reset confirmation

kevinf100 committed:
chore: update epoch's new home

kevinf100 committed:
feat: Added link to source

kevinf100 committed:
chore: comments for logos and fix alt for Github img

PentSec commented:
These are very good ideas, but using `window.confirm` stops the entire workflow in the DOM. The best way to do this is to create a modal-like component. I don't think it's necessary, though. But it's a good idea.

kevinf100 commented:
The website is not getting anything or doing anything in the background so it shouldn't matter much that it's being paused. But when I get home I can just create a simple overlay confirmation instead.
________________________________
From: Jeff. ***@***.***>
Sent: Friday, September 12, 2025 4:35 PM
To: MelEnt/mel-talent-calc ***@***.***>
Cc: kevinf100 ***@***.***>; Mention ***@***.***>
Subject: Re: [MelEnt/mel-talent-calc] Minor changes (PR #2)

[https://avatars.githubusercontent.com/u/11955573?s=20&v=4]PentSec left a comment (MelEnt/mel-talent-calc#2)<https://github.com/MelEnt/mel-talent-calc/pull/2#issuecomment-3286752549>

These are very good ideas, but using window.confirm stops the entire workflow in the DOM. The best way to do this is to create a modal-like component. I don't think it's necessary, though. But it's a good idea.

—
Reply to this email directly, view it on GitHub<https://github.com/MelEnt/mel-talent-calc/pull/2#issuecomment-3286752549>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ADBQBUEUCACCUCXRGC542VL3SMVATAVCNFSM6AAAAACGLE7DO2VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZTEOBWG42TENJUHE>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

kevinf100 commented:
Updated to have a custom Dialog. Made it a little more than simple. I updated the Clipboard copier to also work with the new dialog. Also created some not so great looking pictures for the yes and no buttons.
Now also hosting at.
https://epoch-talents.onlinegameservers.top/
<img width="671" height="177" alt="image" src="https://github.com/user-attachments/assets/d2032214-9e00-4751-947a-202ff40adb25" />

kevinf100 committed:
Feat: Created custom reset Dialog to not hold the DOM
Feat: Also updated the clipboard to work with other elements now.

kevinf100 committed:
feat: If any points are spent and changing classes, ask to change.
fix: changing classes asked if you would like to reset all trees.
chore: resetAll now has an arg to skip asking.
chore: showCustomConfirm is now outside of useTalentTrees and can be called from other places.

kevinf100 commented:
https://github.com/MelEnt/mel-talent-calc/pull/4
